src/actions/config.js

```diff
@@ -82,6 +83,27 @@ export function configDidLoad(config) {
   };
 }
 
+const DEFAULT_I18N_URL = 'https://raw.githubusercontent.com/netlify/netlify-cms/master/src';
```

Comment:
As I mentioned in another comment, we can allow translation objects to be loaded in through the registry and forego any fetching while allowing translation to be independent of cms releases. I believe that would address concerns on both sides, let me know if you think otherwise.

The registry methods are added to the `window` object, source is here: https://github.com/netlify/netlify-cms/blob/1d08f1a33b9562e60145dc9704eb6379f64df044/src/lib/registry.js